util/econftool.c

```diff
@@ -320,41 +317,42 @@ static int econf_edit(void)
     }
 
     /* if path does not exist, create it */
-    if (access(path, F_OK) == -1 && errno == ENOENT) {
-        if (mkdir(path, 0755) != 0) {
+    if (access(conf_dir, F_OK) == -1 && errno == ENOENT) {
```

Comment:
```
    if (mkdir(conf_dir, 0755) != 0 && errno != EEXIST) {
```

Reply:
Hm, this produces every time this ugly error message:
"mkdir() failed: File exists"